src/DragSelect.vue

```diff
@@ -100,15 +106,21 @@
       isItemSelected ({ $el }) {
         if ($el.classList.contains(this.selectorClass)) {
           const boxA = this.selectionBox
-          const boxB = $el.getBoundingClientRect()
-          const elTop = boxB.top + window.scrollY
-          const elLeft = boxB.left + window.scrollX
+          const boxB = {
+            top: $el.offsetTop,
+            left: $el.offsetLeft,
+            width: $el.clientWidth,
+            height: $el.clientHeight,
+          };
+
+          const elTop = boxB.top
```

Comment:
The variables `elTop` and `elLeft` are no longer necessary. Use `boxB.top`/ `boxB.left` directly.